### PR TITLE
DOC: Added page describing how to contribute to the docs team

### PR DESCRIPTION
**Pull request by melissawm**
Sorry, I messed up the git history on the previous PR (#15908 ).

Let me know if there are any other comments. I'm closing the old PR so this is cleaner.

### CODE REVIEW

**Comment by charris** [comment]
No need to apologize. You aren't the first and are unlikely to be the last :) Easy fix and on we go.

File: doc/source/dev/howto-docs.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+other user-generated content. It may include learning-oriented content (such as
+tutorials and how-tos), use-cases or in-depth explanations and reference for
+developers.
+
+If you're reading this page, you probably want to help. This guide is meant to
+help you decide which kind of content you'll write, as well as give you some
+tips and instructions for submitting it to the official NumPy documentation
+(that is, the documentation that ships with NumPy and lives on the
+:ref:`official project pages <numpy_docs_mainpage>`). Keep in mind that if you
+don't want to do this, writing a tutorial on your own blog, creating a youtube
```
Comment (rgommers) [inline doc/source/dev/howto-docs.rst:19]: minor: spelling "YouTube"

File: doc/source/dev/howto-docs.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  documents. They are automatically generated when the NumPy documentation is
+  :ref:`built from source<howto-build-docs>`.
+
+**Datasets**
+
+If you are writing a tutorial or how-to, we encourage you to use real images and
+data (provided they are appropriately licensed and available). This makes the
+material more engaging for readers, and choosing the right data can add
+pedagogical value to your content.
+
```
Comment (rgommers) [inline doc/source/dev/howto-docs.rst:63]: I suggest adding:
```
*Note: currently we cannot easily use data in other packages (except, e.g., from SciPy or Matplotlib).
We plan to create a dedicated datasets package, but that's not ready yet - please discuss with us
if you have data sources in mind.*
```

**Review by rgommers — COMMENTED** [review]
Looks great, thanks @melissawm! Just one substantial comment from me, on the datasets.

**Comment by rgommers** [comment]
LGTM, merged. Thanks @melissawm!

**Pull request merged**